fdbclient/NativeAPI.actor.cpp

```diff
@@ -640,11 +603,12 @@ DatabaseContext::DatabaseContext(Reference<AsyncVar<Reference<ClusterConnectionF
 	monitorMasterProxiesInfoChange = monitorMasterProxiesChange(clientInfo, &masterProxiesChangeTrigger);
 	clientStatusUpdater.actor = clientStatusUpdateActor(this);
 	if (apiVersionAtLeast(630)) {
-		registerSpecialKeySpaceModule(std::make_unique<TransactionModule>(
-		    KeyRangeRef(LiteralStringRef("\xff\xff/transaction/"), LiteralStringRef("\xff\xff/transaction0"))));
-		registerSpecialKeySpaceModule(std::make_unique<WorkerInterfacesSpecialKeyImpl>(KeyRangeRef(
+		registerSpecialKeySpaceModule(SpecialKeySpace::TRANSACTION, std::make_unique<ConflictingKeysImpl>(conflictingKeysRange));
```

Comment:
Make sure (add a test?) that reading [\xff\xff/transaction/, \xff\xff/transaction0) is legal without the relaxed option set, but reading through \xff\xff/transaction0 is not legal

Reply:
I have added tests in the correctness test for these cases.